.env.example

```diff
@@ -1,4 +1,4 @@
 DATABASE_URL="YOUR_DATABASE_URL?schema=prisma"
 SHADOW_DATABASE_URL="YOUR_SHADOW_DATABASE_URL?schema=shadow"
-API_URL="http://localhost:4000"
+REACT_APP_API_URL="http://localhost:4000"
```

Comment:
Nicely figured out, I know that was tricky

Reply:
Yes nice! Let me know when you figure out "process("dotenv").config()"? I think thats the way to get it in express? If I understadn correctly.

Reply:
```
require('dotenv').config()
console.log("My URL", process.env.REACT_APP_API_URL)

```